### PR TITLE
Remove redundant UUID wrapper module, update uuid package to 9.x

### PR DESCRIPTION
**Pull request by j-f1**
I noticed that:

- The web and electron versions of our `uuid` module both looked exactly the same…
- …and deferred to the `uuid` package…
- …and the async version just called the sync API.

So now we will just use the `uuid` package directly everywhere.

### CODE REVIEW

File: packages/desktop-client/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "@types/react-dom": "^18.2.1",
     "@types/react-modal": "^3.16.0",
     "@types/react-router-dom": "^5.3.3",
+    "@types/uuid": "^9.0.2",
```
Comment (MatissJanis) [inline packages/desktop-client/package.json:24]: **💬 suggestion:** ‏you'll also want to add `uuid` to `desktop-client` dependencies now.

File: packages/loot-core/package.json [hunk 2]
```diff
@@ -50,6 +50,7 @@
     "@types/jest": "^27.5.0",
     "@types/jlongster__sql.js": "npm:@types/sql.js@latest",
     "@types/pegjs": "^0.10.3",
+    "@types/uuid": "^9.0.2",
```
Comment (MatissJanis) [inline packages/loot-core/package.json:53]: **❓ question:** ‏we are actually using uuid v3 in Actual. Can we either upgrade it to v9 or downgrade the types to v3? Both of these things should +- match.

File: packages/loot-core/src/platform/client/fetch/index.browser.ts [hunk 3]
```diff
@@ -147,21 +148,21 @@ export const send: T.Send = function (
...
-        undoTag: undo.snapshot(),
-        catchErrors,
-      };
-      if (messageQueue) {
-        messageQueue.push(message);
-      } else {
-        globalWorker.postMessage(message);
-      }
-    });
+    let id = uuid.v4();
```
Comment (MatissJanis) [inline packages/loot-core/src/platform/client/fetch/index.browser.ts:151]: Cool. I like this conversion!

File: packages/loot-core/package.json (hunk 2 above)
Comment (j-f1) [reply packages/loot-core/package.json:53]: Updated to v9!

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**